frontend/src/components/commons/SideNavigation.jsx

```diff
@@ -1,18 +1,82 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
+const styles = {
+  buttonList: {
+    width: 200,
+    paddingRight: 25,
+    float: "left"
+  },
+  button: {
+    width: "100%",
+    marginBottom: 10
+  },
+  bodyContent: {
+    display: "flex",
+    justifyContent: "flext-end"
+  },
+  secondaryButton: {
+    border: "none"
+  },
+  primaryButton: {
+    border: "2px solid #009fae"
+  }
+};
+
 class SideNavigation extends Component {
   static propTypes = {
-    progressSpecs: PropTypes.array.isRequired,
+    navSpecs: PropTypes.array.isRequired,
     currentNode: PropTypes.number.isRequired
   };
 
   state = {
     currentNode: this.props.currentNode,
-    currentBody: this.props.progressSpecs[this.props.currentNode].body
+    currentBody: this.props.navSpecs[this.props.currentNode].body
   };
+
+  changeNode(id) {
+    this.setState({ currentNode: id, currentBody: this.props.navSpecs[id].body });
+  }
+
+  genButtonID(id, status) {
+    return "unit-test-sidenav-button" + id + "-" + status;
+  }
+
   render() {
-    return <div>STUBBED SideNavigation Component</div>;
+    const body_id = this.props.navSpecs[this.state.currentNode].text;
+    return (
+      <div className="side-nav-pane">
```

Comment:
This is my bad, but I don't think you need this className anymore since styles are in objects.

Reply:
fixed in 539ddb3